src/scss/_table.scss

```diff
@@ -0,0 +1,319 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+.table-base {
+    width: 100%;
+    margin: 0;
+    margin-bottom: 1rem;
+    padding: 0;
+
+    table-layout: fixed;
+    background-color: var(--color-bg-dark);
+    border-collapse: collapse;
+}
+
+.table-base thead {
+    border-bottom: 2px solid var(--color-lavender);
+}
+
+.table-base tr {
+    padding: 0.35em;
+
+    border-bottom: 1px solid var(--color-bg-light);
+}
+
+.table-base tr:last-child {
+    border-bottom: none;
+}
+
+.table-base caption {
+    margin: 0.5em 0 0.75em;
+
+    font-size: 1.5em;
+
+    div {
+        display: flex;
+        justify-content: space-between;
+        padding-left: 1rem;
+    }
+}
+
+.table-base th {
+    padding: 0.625em;
+
+    font-weight: bold;
+
+    text-align: left;
+}
+
+.table-base td {
+    padding: 0.625em;
+
+    font-weight: lighter;
+}
+
+.table-base .-line-break {
+    word-wrap: break-word;
+}
+
+/* responsive style applied when screen width is reduced to < 600px */
+@media screen and (max-width: 600px) {
+    .table-base {
+        border: 0;
+    }
+
+    .table-base caption {
+        font-size: 1.3em;
+    }
+
+    .table-base thead {
+        position: absolute;
+
+        width: 1px;
+        height: 1px;
+        margin: -1px;
+        padding: 0;
+        overflow: hidden;
+
+        border: none;
+
+        clip: rect(0 0 0 0);
+    }
+
+    .table-base tr {
+        display: block;
+        margin-bottom: 0.625em;
+
+        border-bottom: 3px solid var(--color-lavender);
+    }
+
+    .table-base td {
+        display: block;
+
+        font-size: 0.8em;
+        text-align: right;
+
+        border-bottom: 1px solid var(--color-bg-light);
+    }
+
+    .table-base td::before {
+        float: left;
+
+        font-weight: bold;
+
+        content: attr(data-label);
+    }
+
+    .table-base td:last-child {
+        border-bottom: 0;
+    }
+}
+
+// end table-base
+
+// style below this should be removed when integrating new table with the rest of the application
+.table {
+    width: 100%;
+    margin: 1rem 0;
+    padding-bottom: 0.5rem;
+
+    font-size: 1rem;
+
+    background: var(--color-bg-dark);
+}
```

Comment:
this style is being move from _main.scss
most of the code below this section will be removed when adapting the hooks page to use the new table design